server/src/test/java/org/elasticsearch/transport/RemoteClusterServiceTests.java

```diff
@@ -163,33 +161,6 @@ public void testBuildRemoteClustersDynamicConfig() throws Exception {
         assertEquals("quux-proxy:19300", map.get("quux").v1());
         assertEquals(quux.getVersion(), Version.CURRENT.minimumCompatibilityVersion());
 
-        assertSettingDeprecationsAndWarnings(new String[]{"search.remote.quux.seeds", "search.remote.quux.proxy"});
-    }
-
-    public void testBuildRemoteClustersDynamicConfigWithDuplicate() {
```

Comment:
I think these were not needed any more now that we don't have both `cluster.remote` and `search.remote` settings to account for but we can reintroduce duplication validation if need be in `RemoteClusterAware.java`

Reply:
correct, we don't need this anymore.